docs/README.md

```diff
@@ -1496,16 +1496,16 @@ mutation {
 }
 ```
 
-## Deploying
+### Serverless ready
```

Comment:
Why renaming this section? For me "Deploying" was clearer, maybe "Deploying your application"?

Reply:
Agree

Reply:
@NickSeagull @javiertoledo, the idea is to have this structure:

- Features
  - Serverless ready (is a feature)
    - Deploying to AWS
    - Deploying to Azure
    - Deploying to a Kubernetes cluster
    - Deploying to Pivotal Cloud Foundry (or whatever cloud supporting functions)

That's the reason for this change. Take into account that the table of content shows headings up to level 3, so the Deploying to AWS section is still quickly visible.

**Having said that, do you still want to change the headings structure?** it is OK for me, I just want to be sure you understand the motivation because for me **the Booster capacity to deploy in any cloud is a big win and a great feature.**

Reply:
I agree with your explanation then, but I'd call the section "Cloud Native" rather than "Serverless ready"